app/src/main/java/org/simple/clinic/storage/Migration_45_46.kt

```diff
@@ -0,0 +1,22 @@
+package org.simple.clinic.storage
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+@Suppress("ClassName")
+class Migration_45_46 : Migration(45, 46) {
```

Comment:
Can you add a test for this migration as well, please?

Reply:
done